src/status_im/accounts/update/core.cljs

```diff
@@ -1,9 +1,46 @@
 (ns status-im.accounts.update.core
   (:require [status-im.data-store.accounts :as accounts-store]
             [status-im.transport.message.protocol :as protocol]
+            [status-im.data-store.transport :as transport-store]
             [status-im.transport.message.contact :as message.contact]
             [status-im.utils.fx :as fx]))
 
+(fx/defn account-update-message [{:keys [db]}]
+  (let [account (:account/account db)
+        fcm-token (get-in db [:notifications :fcm-token])
+        {:keys [name photo-path address]} account]
+    (message.contact/ContactUpdate. name photo-path address fcm-token)))
+
+(fx/defn send-contact-update-fx
+  [{:keys [db] :as cofx} chat-id payload]
+  (when-let [chat (get-in cofx [:db :transport/chats chat-id])]
+    (let [updated-chat  (assoc chat :resend? "contact-update")
+          tx            [(transport-store/save-transport-tx {:chat-id chat-id
+                                                             :chat    updated-chat})]
+          success-event [:transport/contact-message-sent chat-id]]
+      (fx/merge cofx
+                {:db (assoc-in db
+                               [:transport/chats chat-id :resend?]
+                               "contact-update")
```

Comment:
that does the same thing as `let [updated-chat ....` a few lines on top of it. Is there any way to join them?